app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes
+  has_many :comments
+
+  after_create :update_user_posts_counter
+
+  def update_user_posts_counter
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
- [ ] [OPTIONAL]
Following best practices `update_posts_counter` should be marked as _private_ after implementing the `after_create` method.